invenio_rdm_records/services/tasks.py

```diff
@@ -28,3 +30,40 @@ def update_expired_embargos():
         except EmbargoNotLiftedError as ex:
             current_app.logger.warning(ex.description)
             continue
+
+
+@shared_task(ignore_result=True)
+def reindex_stats():
+    """Reindex the documents where the stats have changed."""
+    buffer_index = prefix_index("buffer_stats_to_reindex")
```

Comment:
could you explain me the concept of the buffer ? I think I have never worked with one